File: FaceSample/app/src/main/java/com/regula/facesample/items/customization/OverlayCustomItem.java
package com.regula.facesample.items.customization;

import android.content.Context;

import com.regula.facesample.items.CategoryItem;
import com.regula.facesample.util.FaceCaptureResponseUtil;
import com.regula.facesdk.FaceSDK;

/**
 * Created by Sergey Yakimchik on 24.08.21.
 * Copyright (c) 2021 Regula. All rights reserved.
 */

public class OverlayCustomItem extends CategoryItem {

    @Override
    public void onItemSelected(Context context) {
        // uncomment in res/values/colors.xml:
        // all
        // facesdk_overlay_background_dark
        // facesdk_overlay_background_white
        // facesdk_overlay_border_default
        // facesdk_overlay_border_active
        FaceSDK.Instance().presentFaceCaptureActivity(context, faceCaptureResponse -> FaceCaptureResponseUtil.response(context, faceCaptureResponse));
    }

    @Override
    public String getTitle() {
        return "Custom Overlay";
    }

    @Override
    public String getDescription() {
        return "Change colors for overlay";
    }
}

File: FaceSampleKotlin/app/src/main/java/com/regula/facesamplekotlin/items/customization/OverlayCustomItem.kt
package com.regula.facesamplekotlin.items.customization

import android.content.Context
import com.regula.facesamplekotlin.category.CategoryItem
import com.regula.facesamplekotlin.util.FaceCaptureResponseUtil
import com.regula.facesdk.FaceSDK
import com.regula.facesdk.model.results.FaceCaptureResponse

class OverlayCustomItem : CategoryItem() {

    override fun onItemSelected(context: Context) {
        // uncomment in res/values/colors.xml:
        // all
        // facesdk_overlay_background_dark
        // facesdk_overlay_background_white
        // facesdk_overlay_border_default
        // facesdk_overlay_border_active
        FaceSDK.Instance().presentFaceCaptureActivity(
            context
        ) { faceCaptureResponse: FaceCaptureResponse -> FaceCaptureResponseUtil.response(context, faceCaptureResponse) }
    }

    override val title: String
        get() = "Custom Overlay"

    override val description: String
        get() = "Change colors for overlay"
}